src/_scss/pages/covid19/_stickyHeader.scss

```diff
@@ -0,0 +1,74 @@
+.sticky-header .sticky-header__container .sticky-header__header {
```

Comment:
How reusable are these styles gonna be or are they intended to be? I know this sticky header is gonna be changed throughout the site, how much will the covid19 styles apply to the rest of the site relative to this header?

Wondering if we can add some of this to the main stickyHeader partial...?

I feel pretty disappointed about the current state of our SASS organization etc so just calling that concern out here.

Reply:
Agree, ideally this sort of thing would be in the `layouts` directory. Outside the scope of this PR (but showing up as changed since I broke `covid19/index.scss` into smaller files.

Reply:
Sounds good!